### DISCUSSION TITLE
How to train models for different layouts?

### DISCUSSION THREAD

**Discussion by sheltonsuen** [PaddlePaddle/PaddleOCR#8416] [Q&A]
I'm wondering the best approaches to recognize a card with multiple layouts, 

CARD:  1. latest, 2. legacy ... etc
![image](https://user-images.githubusercontent.com/44127430/203464432-caea00dd-5bac-4341-9c68-785de25a6041.png)
![image](https://user-images.githubusercontent.com/44127430/203464536-207c52e8-3dc4-4a68-ba0f-d4c2427d931c.png)

What I have tried:
1. mege all layout images into one dataset and train it for one model to predict  -- the accuracy is not so good, some field mismatched
2. train models for each layout and add a classfier to dispatch the task to the coorsponding model -- not completed yet

What I want to know:

since I'm a newcomer into this field, i'm not sure if it's right, does anyone have similar situations? or some advice, thanks a lot

**Comment by AkshatHotCode** [comment]
Hi, I am facing some issues with preparing my dataset. Can you please briefly explain how did you prepare your dataset for the same.
Thanks
